
    [global::Xunit.TestCaseOrderer("ScenarioTests.Internal.ScenarioFactTestCaseOrderer", "ScenarioTests")]
    public partial class C
    {
		[global::System.Runtime.CompilerServices.CompilerGenerated]
		[global::System.Diagnostics.DebuggerStepThrough]
		[global::ScenarioTests.Internal.ScenarioFact(DisplayName = "Scenario_Name with a $!@$@!", FactName = "Name with a $!@$@!", ExecutionPolicy = global::ScenarioTests.ScenarioTestExecutionPolicy.EndAfterConclusion, RunInIsolation = true, FileName = "", LineNumber = 5)]
		public void Scenario_Name_with_a_(global::ScenarioTests.ScenarioContext scenarioContext)
		{
			try
			{
				Scenario(scenarioContext);
			}
			catch(global::ScenarioTests.Internal.ScenarioAbortException) { }
		}

    }